docs/source/developer_zone/dev_environment.rst

```diff
@@ -130,6 +130,17 @@ Install development dependencies
     micromamba create -n mamba -c conda-forge -f dev/environment.yml
     micromamba activate -n mamba
 
+.. note::
+
+    You need to have ccache installed on your system.
+    You can either install it via your package manager or via conda-forge
+    in this development environment with:
+
+    .. code:: bash
+
+        micromamba install -n mamba -c conda-forge ccache
+
+
```

Comment:
I was more thinking of a `note` just after talking about presets since it's related (just after this [line](https://github.com/mamba-org/mamba/blob/main/docs/source/developer_zone/dev_environment.rst?plain=1#L141)).
Something like:
```
.. note::
    If you do choose to use the provided CMake Preset, you may need to add ``ccache`` as an extra requirement as specified in ``dev/environment-dev-extra.yml``.
```